plugin/contrib/mesos/pkg/queue/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package queue implements a FIFO queue based on pkg/client/cache/fifo.go@release-0.6
```

Comment:
... implements different kind of queues:
- a priority queue
- a delay queue ordering items by a delay, returning them when the delay has passed
- a historical queue (@jdef: can you give a good short explanation?)